main.go

```diff
@@ -343,7 +349,19 @@ func before(c *cli.Context) (*state.HelmState, helmexec.Interface, error) {
 			log.Printf("err: Cannot use option --kube-context and set attribute context.")
 			os.Exit(1)
 		}
+
+		if st.Helm.Context != "" {
+			log.Printf("err: Cannot set atrribute context on both helmspec and context.")
+			os.Exit(1)
+		}
+
 		kubeContext = st.Context
+	} else if st.Helm.Context != "" {
+		if kubeContext != "" {
+			log.Printf("err: Cannot use option --kube-context and set attribute context.")
+			os.Exit(1)
+		}
```

Comment:
Similarty to https://github.com/roboll/helmfile/pull/171/files#r195421646, I'm fine with this situation, but I prefer the value from the command-line takes precedence